cluster/addons/addon-manager/Makefile

```diff
@@ -15,10 +15,10 @@
 IMAGE=staging-k8s.gcr.io/kube-addon-manager
 ARCH?=amd64
 TEMP_DIR:=$(shell mktemp -d)
-VERSION=v9.0
+VERSION=v9.1
```

Comment:
We usually recommend `v9.0.1` when nothing is changing except the base package versions. Up to you.

Reply:
Thanks Tim, that is better. Updated.